src/modules/AnalysisExecution/components/ViewEditAnalysis/Settings/presenter.jsx

```diff
@@ -59,7 +62,7 @@ function AnalysisSettings(props) {
             modifiers: { uneditable: !isEditable } 
           })}
         >
-            {isEditable ?
+            {isTypeEditable ?
```

Comment:
Why do we now ignore "isEditable"? Shouldn't it be isEditable && "isTypeEditable"?

Reply:
"isEditable" is used in "isTypeEditable" initializtion

Reply:
right, missed that